src/util.js

```diff
@@ -28,6 +28,7 @@
 import stream from 'web-stream-tools';
 import { getBigInteger } from './biginteger';
 
+// eslint-disable-line no-process-env
```

Comment:
If you want it on a separate line I think you have to do
```suggestion
// eslint-disable-next-line no-process-env
```